### ISSUE TITLE
Term 'CSS' in heading breaks heading style

### ISSUE THREAD

**Issue by hArpanet** [selfthinker/dokuwiki_plugin_wrap#75]
When placing the term 'CSS' in a heading, the heading style is no longer rendered but shown in plaintext.

eg. this will not render the heading...
```
<div indent>
=== CSS Stylesheets ===
<div indent>
CSS stylesheets provide the visual design ...
</div>
</div>
```

but this is fine...
```
<div indent>
=== C-S-S Stylesheets ===
<div indent>
CSS stylesheets provide the visual design ...
</div>
</div>
```

**Comment by selfthinker** [comment]
This also happens for entities like "...", "=>", "(c)" etc in headings.
I guess now we know why heading support is still labelled "experimental"... ;-)

I've only looked briefly at this, but it seems to be a quite tricky problem to solve. I hope @Chris--S will be able to help with this.

**Comment by Chris--S** [comment]
Should be resolved by https://github.com/selfthinker/dokuwiki_plugin_wrap/pull/102

**Comment by selfthinker** [comment]
Indeed, resolved by #102. That fix will be available in the next release (probably only a few days away).
